aws-auth-cognito/build.gradle

```diff
@@ -34,6 +34,7 @@ dependencies {
     implementation dependency.aws.cognitoidentityprovider
 
     testImplementation project(path: ':testutils')
+    testImplementation project(path: ':aws-auth-cognito')
```

Comment:
why is this required?

Reply:
AWS Kotlin SDK upgraded the Kotlin version to 1.7.10 and it was resulting in build errors. So we had to upgrade the Kotlin version too. New version requires explicit main compilation dependency on test implementation. Not sure if there is a better way to do this. Please advice.